docs/Using-Docker.md

```diff
@@ -1,6 +1,7 @@
 # Using Docker For ML-Agents (Experimental)
```

Comment:
- Remove "(Experimental)"? If so, then also remove it from the Repo Homepage (features section).
- (Minor) In this page we used 3D Ball as the running example for container and env names. But now the image "docker_build_settings.png" links to the Gridworld scene. I think we ought to either update the naming conventions or have the image point to 3Dball, just for consistency.

Reply:
Both good points.  Will fix.